javascript/javascript-7-oop.md

```diff
@@ -21,7 +21,7 @@
 - [JavaScript 6 - Object Literals Lesson](/javascript/javascript-6-object-literals.md)
 
 ### Motivation
-### Why learn oop in javascript?
+### Why learn OOP in Javascript?
 - It helps to organize larger projects.
 - It support OOP because it supports inheritance through prototyping as well as properties and methods.
 - Here is the tutorial for object-oriented javascript for beginners[https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS]
```

Comment:
Add a label for the link, showing the link is not a good idea.
And make javascript to JavaScript everywhere.